### PR TITLE
Internationalize site with support for English/Spanish

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR adds the [react-i18next](https://github.com/i18next/react-i18next) and [i18next-browser-languagedetector](https://github.com/i18next/i18next-browser-languageDetector) libraries, and creates public/locales/[en|es]/translation.json files for English (en) and Spanish (es), where all content can be centralized.

To view Spanish, check out this branch and visit http://localhost:3000/?lng=es. 

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,108 +1,113 @@
...
-    "terser-webpack-plugin": "^2.2.1",
-    "webpack": "^4.41.2",
-    "webpack-bundle-analyzer": "^3.6.0",
-    "webpack-cli": "^3.3.10",
-    "webpackbar": "^4.0.0"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
+    "name": "cagov-benefits-screener",
```
Comment (kalvinwang) [inline package.json:2]: well, that's annoying. it's the linter though, so this is hopefully a one-off extra-indentation change?

Comment (sharonwarner) [reply package.json:2]: that's really weird... i wonder why this wasn't updated when i ran the linter the first time...

Comment (kalvinwang) [reply package.json:2]: If it changes back when you run the linter I will investigate :p

Comment (kalvinwang) [reply package.json:2]: If it changes back when you run the linter I will investigate :p

**Pull request merged**